cookbooks/cdo-repository/attributes/default.rb

```diff
@@ -4,6 +4,6 @@
   url: 'https://github.com/code-dot-org/code-dot-org.git',
   # Sync to the default branch based on the environment.
   branch: adhoc ? 'staging' : node.chef_environment,
-  # Shallow clone for adhoc environment.
-  depth: adhoc ? 1 : nil
+  # Shallow clone.
+  depth: 1
```

Comment:
Initial deep clone of our repository can take more than ten minutes, which causes timeouts.  We're okay with shallow-cloning on any new stack, not just adhocs, because of this.